Tools/_sunrise/changelog/manual_changelog.py

```
@@ -72,11 +72,11 @@ def flush_changes(changes):
     print("Type or paste in changelog entries from GitHub.\n")
 
     try:
-        # Parse the stream.
+        # Разбираем входной поток.
         for line in stream:
             line = line.strip()
```

Ruff (0.16.1)

[warning] `for` loop variable `line` overwritten by assignment target

(PLW2901)